### DISCUSSION TITLE
3.88.0

### DISCUSSION THREAD

**Discussion by aviramha** [metalbear-co/mirrord#2249] [Announcements]
### Added

- Add log level and log destination for int proxy
  [#2246](https://github.com/metalbear-co/mirrord/issues/2246)

### Changed

- 1. mirrord CLI now does not check target type when the `copy_target` feature
  is enabled. The check is now done only in the operator.
  2. `mirrord operator setup` not includes permissions to read and change
  rollouts scale.

### Fixed

- Incoming traffic was being mirrord when set to `false`.

<hr /><em>This discussion was created from the release <a href='https://github.com/metalbear-co/mirrord/releases/tag/3.88.0'>3.88.0</a>.</em>

**Comment by rcourtman** [comment]


**Reply by aviramha** [reply]
@rcourtman what is this? is this some weird GitHub bug? account hijack? AI?
